src/components/Account.vue

```diff
@@ -38,55 +50,145 @@
         @Emit()
         // tslint:disable-next-line no-empty
         private changed(label: string) {}
+
+        @Watch('image', { immediate: true })
+        private onImageChange() {
+            this.showImage = !!this.image;
+
+            // load clip path if not done so yet
+            if (!this.showImage || document.getElementById('nimiq-hexagon-clip')) return;
+            /* tslint:disable max-line-length */
+            document.body.insertAdjacentHTML('beforeend', `
+                <svg width="0" height="0" viewBox="0 0 146 146">
+                    <defs>
+                        <clipPath id="nimiq-hexagon-clip" clipPathUnits="objectBoundingBox">
+                            <path d="M.302.055A.106.106 0 0 0 .21.108l-.196.34a.106.106 0 0 0 0 .105l.196.34a.106.106 0 0 0 .092.052h.392c.038 0 .073-.02.092-.053l.196-.34a.106.106 0 0 0 0-.105L.786.107A.106.106 0 0 0 .694.056z">
+                            </path>
+                        </clipPath>
+                    </defs>
+                </svg>
+            `);
+            /* tslint:enable max-line-length */
+        }
     }
 </script>
 
 <style scoped>
     .account {
-        width: 100%;
         display: flex;
-        flex-direction: row;
         align-items: center;
         justify-content: space-between;
         padding: 1.75rem 2rem;
         box-sizing: border-box;
         flex-shrink: 0;
         font-size: 2rem;
-        line-height: 1.25;
+        line-height: 1.2;
+    }
+
+    .account.row {
+        width: 100%;
+        flex-direction: row;
+    }
+
+    .account.column {
+        flex-direction: column;
     }
 
     .identicon-and-label {
         display: flex;
-        flex-direction: row;
         align-items: center;
+    }
+
+    .row .identicon-and-label {
+        flex-direction: row;
         overflow: hidden;
         min-width: 5.625rem;
     }
 
+    .column .identicon-and-label {
+        flex-direction: column;
```

Comment:
This would then also need a `width: 100%;` to make the Identicon not "wobble" when changing the label.